static/app/views/performance/transactionSummary/transactionSpans/index.spec.tsx

```diff
@@ -142,29 +141,12 @@ describe('Performance > Transaction Spans', function () {
         const grid = await screen.findByTestId('grid-editable');
         expect(await within(grid).findByText('Span Operation')).toBeInTheDocument();
         expect(await within(grid).findByText('Span Name')).toBeInTheDocument();
-        expect(await within(grid).findByText('Span Count')).toBeInTheDocument();
         expect(await within(grid).findByText('Frequency')).toBeInTheDocument();
         expect(await within(grid).findByText(label)).toBeInTheDocument();
-        expect(await within(grid).findByText('Total Self Time')).toBeInTheDocument();
+        expect(await within(grid).findByText('Total Self Time %')).toBeInTheDocument();
```

Comment:
I think maybe the reason we didn't do this the first time around is that self exclusive time summed via the array can be in parallel which makes the sum(exclusive) larger than the duration? Might want to check this since I can't recall.

Reply:
ah that's a good point. I'll make a note of that and surface to the team 👍 IMO it shouldn't be deal breaker for this PR but I'll sort it out asap